analyzer.go

```diff
@@ -43,8 +44,13 @@ func (a *Analyzer) Analyze() (platform.AnalyzedMetadata, error) {
 		appMeta = platform.LayersMetadata{}
 	}
 
+	cacheMeta, err = retrieveCacheMetadata(a.Cache, a.Logger)
+	if err != nil {
+		return platform.AnalyzedMetadata{}, err
+	}
```

Comment:
Don't we want to put it under the if condition in line 52?

Reply:
Good point! I'll make this change.